src/components/buttons/secondary-button/secondary-button.spec.js

```diff
@@ -71,6 +72,26 @@ describe('rendering', () => {
     });
   });
   describe('when using "linkTo"', () => {
+    /* eslint-disable no-console */
+    let log;
+    beforeEach(() => {
+      log = console.error;
+      console.error = jest.fn();
+    });
+    afterEach(() => {
+      console.error = log;
+    });
```

Comment:
Thought for a moment we could `spyOn` and `mockRestore` but I guess that's not what we want.